lib/awesome_nested_set/model/validatable.rb

```diff
@@ -52,6 +55,13 @@ def each_root_valid?(roots_to_validate)
             end
           end
 
+          def left_and_right_within_range?
+            max_node_value = count * 2
+            select("#{scope_string.chomp(", ")}").
+              where("#{quoted_left_column_full_name} > :max_node_value OR #{quoted_right_column_full_name} > :max_node_value", max_node_value: max_node_value).
+              first.nil?
```

Comment:
```suggestion
            !select("#{scope_string.chomp(", ")}").
              where("#{quoted_left_column_full_name} > :max_node_value OR #{quoted_right_column_full_name} > :max_node_value", max_node_value: max_node_value).
              exists?
```

Reply:
Thank you for review @parndt I've addressed the suggested change.